setTRIMDAC_MASK_FROM_FILE.py

```diff
@@ -22,11 +24,14 @@
 print testSuite.chipIDs
 
 for port in testSuite.presentVFAT2sSingle:
-    try:
-        trimDACfileList = open("TrimDACfiles.txt",'r')
-    except:
-        print "No TrimDACfiles.txt to specify paths to TRIM_DACS"
-        break
+    if len(trimfilelist) < 2:
```

Comment:
Rather than this, set ```TrimDACfiles.txt``` as the default to this option, rather than the empty string ```""```